frontend/src/components/partners/partnersActivity.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import { OHSOME_STATS_BASE_URL } from '../../config';
+import { FormattedMessage } from 'react-intl';
+
+import ReactPlaceholder from 'react-placeholder';
+import PartnersProgresBar from './partnersProgresBar';
+import messages from './messages';
+
+export const Activity = (partner) => {
```

Comment:
@CristianAmici this component is raising an error, I think it's because partner is an object with a `partner` propert and you have to use `({ partner })` instead of `(partner)`, like you did in `partnerResources.js`.

CC @dgquail